src/Compiler/FSharp.Compiler.Service.fsproj

```diff
@@ -361,12 +361,15 @@
     <Compile Include="Checking\CheckBasics.fs" />
     <Compile Include="Checking\CheckRecordSyntaxHelpers.fsi" />
     <Compile Include="Checking\CheckRecordSyntaxHelpers.fs" />
-    <Compile Include="Checking\CheckExpressions.fsi" />
-    <Compile Include="Checking\CheckExpressions.fs" />
+    <Compile Include="Checking\Expressions\CheckExpressionsOps.fs" />
+    <Compile Include="Checking\Expressions\CheckExpressions.fsi" />
+    <Compile Include="Checking\Expressions\CheckExpressions.fs" />
```

Comment:
This move is going to break a bunch of docs and benchmarks, relying on hardcoded path.
Consider searching "CheckExpressions" across the codebase, should be easy to fix.

Reply:
Don't think we mention it in the docs. In the benchmarks thouhg we should pin revision in future, I will fix whatever I will find.

Reply:
We mention it in the compiler overview. I am reading it every day recently :)

Reply:
Updated whatever I found.

Reply:
Benchmarks compile and smoke tests pass, that should be good enough, thanks.